solhint.config.js

```diff
@@ -1,16 +1,22 @@
 const customRules = require('./scripts/solhint-custom');
 
 const rules = [
-  'no-unused-vars',
+  'avoid-tx-origin',
   'const-name-snakecase',
   'contract-name-camelcase',
   'event-name-camelcase',
+  'explicit-types',
```

Comment:
We need to bump `solhint` to 3.5.1 if we want to support this change according to [its page](https://protofire.github.io/solhint/docs/rules/best-practises/explicit-types.html#version).